mne/fixes.py

```diff
@@ -52,7 +52,9 @@ def _compare_version(version_a, operator, version_b):
     except ImportError:
         from distutils.version import LooseVersion as parse
 
-    return eval(f'parse("{version_a}") {operator} parse("{version_b}")')
+    with warnings.catch_warnings(record=True):
+        warnings.simplefilter('ignore')
+        return eval(f'parse("{version_a}") {operator} parse("{version_b}")')
```

Comment:
`parse` emitted a warning because some package had a non-PEP440-compliant version. We should hide these from the user, they shouldn't hear about this issue. (It didn't break the comparison in this case anyway -- I think it was a `.develop` or something at the end.)